### PR TITLE
Temporary fix for capabilities with Kernel <4.14

### PR DESCRIPTION
**Pull request by vboulineau**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: Dockerfiles/cluster-agent/arm64/Dockerfile [hunk 1]
```diff
@@ -43,9 +43,9 @@ COPY --from=builder /output /
 # And we need to do it here because capabilities set in `builder` stage are not kept
 # by the `COPY --from=builder`.
 RUN apt-get update \
-    && apt-get install -y libcap2-bin \
-    && setcap cap_net_bind_service+ep /opt/datadog-agent/bin/datadog-cluster-agent \
-    && apt-get remove -y libcap2-bin \
+    && apt-get install -y attr \
+    && setfattr --name="security.capability" --value="0sAQAAAgAEAAAAAAAAAAAAAAAAAAA=" /opt/datadog-agent/bin/datadog-cluster-agent \
```
Comment (clamoriniere) [inline Dockerfiles/cluster-agent/arm64/Dockerfile]: could you add a comment that explains the meaning of this value?

File: Dockerfiles/cluster-agent/amd64/Dockerfile [hunk 2]
```diff
@@ -43,9 +43,9 @@ COPY --from=builder /output /
 # And we need to do it here because capabilities set in `builder` stage are not kept
 # by the `COPY --from=builder`.
 RUN apt-get update \
-    && apt-get install -y libcap2-bin \
-    && setcap cap_net_bind_service+ep /opt/datadog-agent/bin/datadog-cluster-agent \
-    && apt-get remove -y libcap2-bin \
+    && apt-get install -y attr \
+    && setfattr --name="security.capability" --value="0sAQAAAgAEAAAAAAAAAAAAAAAAAAA=" /opt/datadog-agent/bin/datadog-cluster-agent \
```
Comment (clamoriniere) [inline Dockerfiles/cluster-agent/amd64/Dockerfile]: could you add a comment that explains the meaning of this value?

File: Dockerfiles/cluster-agent/arm64/Dockerfile (hunk 1 above)
Comment (vboulineau) [reply Dockerfiles/cluster-agent/arm64/Dockerfile]: Done

**Review by clamoriniere — APPROVED** [review]

**Comment by clamoriniere** [comment]
@vboulineau 
Could you cross reference the initial issue in the description of this PR

**Comment by vboulineau** [comment]
In fact, it wont't fix the issue as the kernel is check is implement in `setxattr()` so the `setfattr` function is also impacted

**Pull request closed without merging**